weblate/static/style-bootstrap.css

```diff
@@ -602,20 +602,26 @@ form + .login-label {
   color: #bfc3c7;
 }
 
+#suggestions {
+  margin-top: 55px;
+}
+
 .btn-float {
+  display: flex;
+  gap: 5px;
+  padding: 4px;
   background-color: #fff;
   box-shadow: 0 2px 10px 0 rgba(0, 0, 0, 0.1);
-  border-radius: 3px;
+  border-radius: 5px;
   position: relative;
-  top: -5px;
+  transform: translateY(-100%);
   z-index: 10;
-  padding: 0 2px;
 }
 
-.btn-float .btn,
-.btn-padded .btn {
```

Comment:
Why `btn-padded` is removed?

Reply:
It applies smaller padding to the buttons than the default, it causes users to miss click. When removed the buttons reset to their default padding which looks better.

Reply:
It applies smaller padding to the buttons than the default, it causes users to miss the click. When removed the buttons reset to their default padding which looks better.